docs/features/records.md

```diff
@@ -108,35 +174,44 @@ The programmer may provide an explicit constructor with the same signature as th
 At runtime the primary constructor
 
 * initializes compiler-generated backing fields for the properties corresponding to the value parameters (if these properties are compiler-provided; [see 1.1.2](#1.1.2)); then
-
 * executes the instance field initializers appearing in the *class-body*; and then
-
 * invokes a base class constructor:
+	* If there are arguments in the *record_base_arguments*, a base constructor selected by overload resolution with these arguments is invoked;
+	* Otherwise a base constructor is invoked with no arguments.
+* executes the body of each *primary_constructor_body*, if any, in source order.
```

Comment:
What's the `primary_constructor_body` is for a non-`abstract`, non-`sealed` base record.